ui/lib/utils/tableColumnFactory.tsx

```diff
@@ -17,16 +17,20 @@ import {
   IColumnKeys,
 } from '@lib/components'
 
-type Bar<T> = { [key: string]: keyof T }
-type BarsConfig<T> = {
-  displayTransKey?: string // it is same as avg field name default
-  bars: [Bar<T>, Bar<T>, Bar<T>?] // [avg, max, min?]
+type Bar = {
```

Comment:
Just found this type's name is the same as the imported component, can we change a name for it, for example: `interface IBar`. others LGTM.